CMakeLists.txt

```diff
@@ -208,10 +208,10 @@ install(
 )
 
 install(PROGRAMS tests/sandbox/sandbox.sh DESTINATION bin)
-install(FILES src/runtime_config/default/actions.js DESTINATION bin)
-install(FILES src/runtime_config/default/validate.js DESTINATION bin)
-install(FILES src/runtime_config/sandbox/resolve.js DESTINATION bin)
-install(FILES src/runtime_config/default/apply.js DESTINATION bin)
+install(FILES samples/constitutions/default/actions.js DESTINATION bin)
```

Comment:
`samples/constitution` or `samples/constitutions`? I went with the latter because:
- Consistency with the other folders under `samples`, which are mostly pluralised
- The subfolders are mostly distinct, named constitutions